addons-community.yaml

```diff
@@ -257,7 +257,7 @@ description: Setup/Manage Apache Camel in your project
 url: http://fabric8.io/guide/forge.html
 website: http://fabric8.io/guide/forge.html
 installCmd: addon-install --coordinate io.fabric8.forge:camel
-repo: https://github.com/fabric8io/fabric8.git
+repo: https://github.com/fabric8io/fabric8-forge.git
 path: /forge/addons/camel
```

Comment:
Path needs to be changed to /addons/camel, since /forge no longer exists in this repository